core/templates/services/i18n-language-code.service.ts

```diff
@@ -53,6 +55,28 @@ export class I18nLanguageCodeService {
         this.getCurrentI18nLanguageCode()) !== -1);
   }
 
+  // Remove this method when translation service is extended.
+  getClassroomTranslationKey(classroomName: string): string {
+    return `I18N_CLASSROOM_${classroomName.toUpperCase()}_TITLE`;
+  }
+
+  // Remove this method when translation service is extended.
+  getTopicTranslationKey(topicId: string, isDescription: boolean): string {
```

Comment:
This seems odd -- make two methods instead, or change isDescription to an enum describing what property of a topic is being considered.

Reply:
So yeah, I decided to make 2 functions separately for titles and descriptions.

Reply:
You may find that the alternative is more generalizable when you extend to include stories etc., but it's up to you.

Reply:
Yeah, created a enum for it now.